src/main/java/org/kairosdb/core/http/rest/MetricsResource.java

```diff
@@ -439,65 +447,68 @@ public Response get(@Context HttpHeaders httpHeaders, String json) throws Except
 		checkNotNull(json);
 		logger.debug(json);
 
-		Span span = createSpan("datapoints_query", httpHeaders);
+		final Span span = createSpan("datapoints_query", httpHeaders);
+		ExecutorService executor = new ThreadPoolExecutor(0,4,0l,TimeUnit.MILLISECONDS, new SynchronousQueue<>(),
```

Comment:
Is 4 enough for the thread pool size?

Reply:
that's a test..will be replaced with env vars later